<launch>
<group ns="quadrotor"> <!-- ROBOT name -->
  <node name="planner" pkg="labrom_mav_planner" type="planner_node">
    <remap from="key_down"   to="/keyboard/key_down"/>
    <remap from="key_up" to="/keyboard/key_up"/>
    <remap from="trajectory" to="trajectory"/>
  </node>
</group>
</launch>